lib/Helper.ts

```diff
@@ -2,7 +2,15 @@ import { Base64 } from '../enum/Dialogflow';
 
 export const getError = (error: any) => {
     if (typeof error === 'object') {
-        return JSON.stringify(error);
+        const errorObject = Object.getOwnPropertyNames(error).reduce((acc, key) => { acc[key] = error[key]; return acc; }, {});
+        return JSON.stringify(errorObject);
+    }
+    return error;
+};
+export const getErrorMessage = (error: any) => {
+    if (typeof error === 'object') {
+        const errorObject = Object.getOwnPropertyNames(error).reduce((acc, key) => { acc[key] = error[key]; return acc; }, {});
+        return JSON.stringify(errorObject['message'].split(/\n/)[1]);
```

Comment:
@ItzzRitik I truncated the message here because Dialogflow messages are too verbose for our needs right now.  We still have the content of the full message in other logs it seems....  so I think we will go with this for now.

So it looks good, and I will merge.  Thanks!

Reply:
@ear-dev Thanks for making this change
This looks good to me